packages/app/src/stores/ConfigStore/ConfigStore.ts

```diff
@@ -25,7 +25,7 @@ const ConfigStore = types
 
         Object.entries(appVariablesOverrides).forEach((entry) => {
           const [key, value] = entry;
-          appVariables[key as keyof AppConfigInterface] = value as any;
+          appVariables[key as keyof AppConfigInterface] = value as string;
```

Comment:
actually this is wrong. It should be 
```suggestion
          appVariables[key as keyof AppConfigExtendedInterface] = value as any;
```

as appVariables are of type `AppConfigExtendedInterface`

Reply:
It doesn't really matter - it's just casting of stuff and doesn't affect anything. And `any` will give us warning.

Reply:
And it gives error `Type 'any' is not assignable to type 'never'.` if AppConfigExtendedInterface is used, whatever the hell it means. Lets just leave it as is ;)